app/views/applications/_show.html.haml

```diff
@@ -11,7 +11,13 @@
       Email
       %small (required)
     %td= @user.email
-
+  %tr
+    %td.left
+      Pronouns
+      %small (optional)
+      %p.small *This does not affect your application.
```

Comment:
Here's a suggested edit, both text and styling. (Since Double Union is an organization and not a person, it seems a bit odd to me to say that Double Union itself respects pronouns; I'd rather say that our members want to use the right pronouns.)

```      
      %small
        Optional — this does not affect your application.
        Double Union members want to use your correct pronouns.
```

![screen shot 2016-12-30 at 11 33 56 pm](https://cloud.githubusercontent.com/assets/978428/21576521/9b4417a0-cee8-11e6-9c2b-e9828b942542.png)

---

Here's what it looks like as-is, where the 'Double Union respects what pronouns you use" isn't inside of the small paragraph and is thus real big:

![screen shot 2016-12-30 at 11 34 12 pm](https://cloud.githubusercontent.com/assets/978428/21576522/a91f0d08-cee8-11e6-84dd-5438563dcd95.png)